pontoon/administration/forms.py

```diff
@@ -61,8 +61,9 @@ def __init__(self, *args, **kwargs):
     def save(self, *args, **kwargs):
         obj = super(ProjectForm, self).save(*args, **kwargs)
         if "disabled" in self.changed_data or "visibility" in self.changed_data:
-            for locale in self.locales.all():
-                locale.aggregate_stats()
+            for project_locale in obj.project_locale.all():
+                project_locale.locale.aggregate_stats()
```

Comment:
I don't understand this. Why did you change locale to project_locale? Stats of the project_locale shouldn't change when you change disabled or visibility flags.

Reply:
As we discussed IRL: this code executes `aggregate_stats()` on locales assigned to a project.